### PR TITLE
change title color according to new branding

### PR DESCRIPTION
**Pull request by xavier**
Love the purple, good job of the marketing team and @degorsse

### CODE REVIEW

**Comment by degorsse** [comment]
well received thanks

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -83,3 +83,7 @@ h1, h2 {
   color: #ccc;
   text-align: center;
 }
+
+#brand {
+  color: purple;
```
Comment (degorsse) [inline app/assets/stylesheets/application.css]: Red would be better

Comment (xavier) [inline app/assets/stylesheets/application.css]: oops, gonna fix it asap, @degorsse what red should I use?

Comment (degorsse) [inline app/assets/stylesheets/application.css]: #DB0000

**Pull request merged**